dotnet/test/AutoGen.Anthropic.Tests/AnthropicClientAgentTest.cs

```diff
@@ -28,4 +31,32 @@ public async Task AnthropicAgentChatCompletionTestAsync()
         await singleAgentTest.UpperCaseTestAsync(agent);
         await singleAgentTest.UpperCaseStreamingTestAsync(agent);
     }
+
+    [ApiKeyFact("ANTHROPIC_API_KEY")]
+    public async Task AnthropicAgentTestProcessImageAsync()
+    {
+        var client = new AnthropicClient(new HttpClient(), AnthropicConstants.Endpoint, AnthropicTestUtils.ApiKey);
+        var agent = new AnthropicClientAgent(
+            client,
+            name: "AnthropicAgent",
+            AnthropicConstants.Claude3Haiku).RegisterMessageConnector();
+
+        var base64Image = await AnthropicTestUtils.Base64FromImageAsync("square.png");
+        var imageMessage = new ChatMessage("user",
```

Comment:
Can you add tests for the following message types
- ImageMessage
- MultiModaMessage with one ImageMessage

Reply:
You can refer to this code on how to do it
- https://github.com/microsoft/autogen/blob/1e5c03f66a0df121642acb277db3e72f52c8a318/dotnet/test/AutoGen.Ollama.Tests/OllamaAgentTests.cs#L117

Reply:
Updated the code, thanks for the sample !